rome-modules/src/main/java/com/rometools/modules/itunes/io/ITunesGenerator.java

```diff
@@ -141,7 +141,7 @@ public void generate(final Module module, final Element element) {
 
         if (itunes.getImage() != null) {
             final Element image = generateSimpleElement("image", "");
-            image.setAttribute("href", itunes.getImage().toExternalForm());
+            image.setAttribute("href", itunes.getImageUri().toString());
```

Comment:
`NullPointerException` is thrown here if image was set the old way 😞

Reply:
@adelnizamutdinov Oops! Thank you for reporting! I merged a fix #448 and released version `1.13.1` - it should become available in maven within several hours!

Reply:
that was quick! Thank you so much